bitrise.yml

```diff
@@ -1406,6 +1406,7 @@ trigger_map:
 - push_branch: v105.0
   workflow: SPM_Deploy_Prod_Beta
 - pull_request_target_branch: main
-  pipeline: pipeline_multiple_shards
+  #pipeline: pipeline_multiple_shards
+  workflow: RunSmokeXCUITestsiPad
```

Comment:
⚠️ revert change once the workflow is tested on bitrise.

Reply:
@isabelrios Could you please kindly start the test on bitwise? I have already approve the change on `test_plan:`. I may not have permission to start a test run on bitwise.

Reply:
@clarmso I can show you how to do that, you should have rights to do it.